app/decorators/voice_out_trunk_decorator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class VoiceOutTrunkDecorator < ResourceDecorator
+  def display_allowed_sip_ips
+    model.allowed_sip_ips.map(&:to_s).join(', ')
+  end
+
+  def display_allowed_rtp_ips
+    model.allowed_rtp_ips ? model.allowed_rtp_ips.map(&:to_s).join(', ') : '0.0.0.0/0'
```

Comment:
if there are no rtp_ips present than it allows all ips not only ipv4, i suggest to show nothing in this case